geolocation/src/definitions.ts

```diff
@@ -180,6 +180,17 @@ export interface PositionOptions {
    * @since 1.0.0
    */
   maximumAge?: number;
+
+    /**
+   * The minumum update interval for location updates.
+   * 
+   * If location updates are available faster than this interval then an update 
+   * will only occur if the minimum update interval has expired since the last location update.
+   * 
```

Comment:
We should document that it is for Android only unless we are also adding this feature to iOS

Reply:
Yeah good point. Already added the note as we agreed.